node/src/components/chain_synchronizer/operations.rs

```diff
@@ -1339,10 +1339,11 @@ impl BlockSignaturesCollector {
     ) -> bool {
         self.0.as_ref().map_or(false, |sigs| {
             are_signatures_sufficient_for_sync_to_genesis(
-                consensus::check_sufficient_finality_signatures(
+                consensus::check_sufficient_finality_signatures_with_quorum_formula(
                     validator_weights,
                     finality_threshold_fraction,
                     sigs,
+                    std::convert::identity,
```

Comment:
Personally, I would simply create a method with meaningful name and the comment that calls `check_sufficient_finality_signatures_with_quorum_formula` with this additional argument.

Reply:
Updated in b898d26fa8e0976231d87894af453a2aa330b8ac